pkg/quarterdeck/mock/quarterdeck.go

```diff
@@ -132,6 +141,13 @@ func UseStatus(status int) HandlerOption {
 	}
 }
 
+// Configure a basic error reply to be returned by the handler
+func UseError(err string) HandlerOption {
+	return func(o *handlerOptions) {
+		o.err = err
+	}
+}
```

Comment:
The Quarterdeck client was returning 500 if no response body was present, so I added this argument to the mock to make sure we can test that statuses are correctly being passed through in tests.